common/Log.hpp

```diff
@@ -311,69 +311,76 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
     do                                                                                             \
     {                                                                                              \
         auto& log_ = Log::logger();                                                                \
+        auto& name_ = log_.name();                                                                 \
         if (LOG_CONDITIONAL(log_, trace))                                                          \
         {                                                                                          \
-            LOG_BODY_(log_, TRACE, "TRC", X, logPrefix, LOG_END);                                  \
+            LOG_BODY_(log_, name_, TRACE, "TRC", X, logPrefix, LOG_END);                           \
         }                                                                                          \
     } while (false)
 
 #define LOG_TRC_NOFILE(X)                                                                          \
     do                                                                                             \
     {                                                                                              \
         auto& log_ = Log::logger();                                                                \
+        auto& name_ = log_.name();                                                                 \
         if (LOG_CONDITIONAL(log_, trace))                                                          \
         {                                                                                          \
-            LOG_BODY_(log_, TRACE, "TRC", X, logPrefix, LOG_END_NOFILE);                           \
+            LOG_BODY_(log_, name_, TRACE, "TRC", X, logPrefix, LOG_END_NOFILE);                    \
         }                                                                                          \
     } while (false)
 
 #define LOG_DBG(X)                                                                                 \
```

Comment:
I'm far from happy with the degree of duplication here - though it pre-dates you.

We should be able to macro generate the body of all these LOG_DBG, LOG_TRC macros with another level of macro wrapper - and make this far more maintainable.

Can you squash this patch into the other large scale change here - and do that simplificaiton ? =)